hudi-flink/src/main/java/org/apache/hudi/streamer/FlinkStreamerConfig.java

```diff
@@ -155,7 +156,11 @@
     conf.setString(FlinkOptions.RECORD_KEY_FIELD, config.recordKeyField);
     conf.setString(FlinkOptions.PARTITION_PATH_FIELD, config.partitionPathField);
     conf.setString(FlinkOptions.KEYGEN_CLASS, config.keygenClass);
-    conf.setInteger(FlinkOptions.WRITE_TASKS, config.writeTaskNum);
+    if (!StringUtils.isNullOrEmpty(config.keygenClass)) {
+      conf.setString(FlinkOptions.KEYGEN_CLASS, config.keygenClass);
+    } else {
+      conf.setString(FlinkOptions.KEYGEN_TYPE, config.keygenType);
+    }    conf.setInteger(FlinkOptions.WRITE_TASKS, config.writeTaskNum);
```

Comment:
`conf.setInteger(FlinkOptions.WRITE_TASKS, config.writeTaskNum);` should start a newline

Reply:
s.g., addressed~